compredict/client.py

```diff
@@ -200,41 +197,58 @@ def get_algorithm(self, algorithm_id: str) -> Union[resources.Algorithm, bool]:
         response = self.connection.GET('/algorithms/{}'.format(algorithm_id))
         return self.__map_resource('Algorithm', response)
 
-    def __process_data(self, data, content_type=None, compression=None):
+    @staticmethod
+    def __raise_error_if_file_type_incorrect(path_to_file: str, type_of_file: str):
+        """
+        Features file can be only provided as .parquet, whereas parameters file can be only provided as .json.
+        This method will raise ValueError if features/parameter file specified, breaks this rule.
+        """
+        split_path = path_to_file.split(".")
```

Comment:
There is a better to extract the extensions in python use
filename, extension = os.path.splitext(path_to_file)